### DISCUSSION TITLE
API Rules for when a field is N/A

### DISCUSSION THREAD

**Discussion by Artur-Galstyan** [pocketbase/pocketbase#2203] [Q&A] [answered]
I have a collection "clips", which also contains a user field, which can also be empty.

I want to create a rule like this

user.id = "" || (@request.auth.id = user.id)

Basically, the idea is that a user should be able to view a clip it it belongs to the user, or if it belongs to no user at all.  

but I get a 404 error. I'm on version 0.14.0.

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, this is a side effect of the recent `COALESCE` removal optimizations. I'll see how to handle it better in the next release.

For now, the above should work if you change `user.id` to just `user`.

**Comment by Artur-Galstyan** [comment]
For reference: on version 0.13.4 it still worked.

**Comment by Artur-Galstyan** [comment]
Thanks, for the swift response :) It worked

**Reply by ganigeorgiev** [reply]
@Artur-Galstyan Just to let you know that I've added a `COALESCE` fallback in [v0.14.2 release](https://github.com/pocketbase/pocketbase/releases/tag/v0.14.2) so now both `user` and `user.id` should work as before.
